Ticket - 27086 Add Cust, Vend, GL, and AR Invoices to Master Importer

sewa67 opened the pull request:
Please review this change and merge into development branch 
Files - util/ImportCust.p,util/impCust.p

sewa67 committed:
Ticket - 27086 Add Cust, Vend, GL, and AR Invoices to Master Importer
Files - util/ImportCust.p,util/impCust.p

brad-advantzware reviewed (changes requested):
Do not merge. This looks OK, but the files he used as template are not current

sewa67 committed:
Ticket -27086 Add Cust, Vend, GL, and AR Invoices to Master Importer
Files - util/ttImport.i,util/ImportCust.p,util/impcust.p,util/ImportGL.p,util/impAR.p,util/importAR.p,util/impGL.p,
             util/impVend.p,util/importVend.p

brad-advantzware committed:
Merge branch 'develop' into feature/27086_Add_Cust_Vend_GL_and_AR_Invoices_to_Master_Importer
